### PR TITLE
#214

### PR DESCRIPTION
**Pull request by kaaleksandr**
Added a property for out type output

### CODE REVIEW

File: krabs/krabs/property.hpp [hunk 1]
```diff
@@ -64,9 +64,17 @@ namespace krabs {
          */
         _TDH_IN_TYPE type() const;
 
+        /**
+         * <summary>
+         * Retrieves the Tdh type of the property.
```
Comment (swannman) [inline krabs/krabs/property.hpp:69]: consider `Retrieves the Tdh out type of the property.` to disambiguate `out_type` from `type` above

**Review by swannman — CHANGES_REQUESTED** [review]
Looks good, I added one minor comment. If you need updated nugets published so you can consume this change, would you mind bumping the nuget versions in the three `.nuspec` files at the root of the repo? Thanks!

**Comment by kaaleksandr** [comment]
> Looks good, I added one minor comment. If you need updated nugets published so you can consume this change, would you mind bumping the nuget versions in the three `.nuspec` files at the root of the repo? Thanks!

Yes of course

**Review by swannman — APPROVED** [review]

**Comment by swannman** [comment]
@kaaleksandr ready to merge once you accept the CLA above. Thanks!

**Comment by kaaleksandr** [comment]
@microsoft-github-policy-service agree

**Pull request merged**